images/win/scripts/Installers/Install-Haskell.ps1

```diff
@@ -13,20 +26,14 @@ $VersionsList = $LatestMajorMinor | ForEach-Object { $_.Group | Select-Object -F
 # The latest version will be installed as a default
 ForEach ($version in $VersionsList)
 {
-     # 9.4.3 has failed choco builds, replace with 9.4.2 for the time being
```

Comment:
We're still using choco to get the available versions (a few lines up). We should use GHCup now.